DEBUG: Debug üzenet
INFO: Info üzenet
WARNING: Warning üzenet
ERROR: Error üzenet
CRITICAL: Critical üzenet
ERROR: Exception üzenet
Traceback (most recent call last):
  File "h:\DADY\rend\konyvtarak\2024\241106_1319_python_logging\python_logging\logging_2.py", line 62, in main
    1 / 0
    ~~^~~
ZeroDivisionError: division by zero
DEBUG: Debug üzenet
INFO: Info üzenet
WARNING: Warning üzenet
ERROR: Error üzenet
CRITICAL: Critical üzenet
ERROR: Exception üzenet
Traceback (most recent call last):
  File "h:\DADY\rend\konyvtarak\2024\241106_1319_python_logging\python_logging\logging_2.py", line 61, in main
    1 / 0
    ~~^~~
ZeroDivisionError: division by zero
